CODEOWNERS

```diff
@@ -1 +1 @@
-* @GoogleCloudPlatform/blueprint-solutions @donmccasland
+* @GoogleCloudPlatform/blueprint-solutions @GoogleCloudPlatform/torus-dpe @GoogleCloudPlatform/dee-platform-ops
```

Comment:
This is managed authoritatively and will be overwritten by our automation. Could you send a PR to update it there?

https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/ae41c774d4e20e303785ebe1554fb040b154d691/infra/terraform/test-org/org/locals.tf#L543